### PR TITLE
feat: [FFM-6680]

### PR DESCRIPTION
**Pull request by crystal-fong**
new resources needed to migrate `useGovernance` hook and FlagSettings component

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/75-cf/pages/FFUIApp/FFUIApp.tsx [hunk 1]
```diff
@@ -52,9 +55,16 @@ const FFUIApp: FC = () => (
...
       usePreferenceStore
     }}
-    customComponents={{ RbacOptionsMenuButton, ContainerSpinner, Description }}
+    customComponents={{
+      ContainerSpinner,
+      Description,
+      EvaluationModal,
+      FeatureWarningTooltip,
+      RbacOptionsMenuButton,
+      RBACTooltip
```
Comment (knagurski) [inline src/modules/75-cf/pages/FFUIApp/FFUIApp.tsx:64]: Nice attention to detail with the ordering

**Review by knagurski — APPROVED** [review]
Looks good 👍

File: src/modules/75-cf/pages/FFUIApp/FFUIApp.tsx (hunk 1 above)
Comment (crystal-fong) [reply src/modules/75-cf/pages/FFUIApp/FFUIApp.tsx:64]: TY, I might open up a chore to put them all in alphabetical order

Comment (knagurski) [reply src/modules/75-cf/pages/FFUIApp/FFUIApp.tsx:64]: I'd normally say to just have a look when you're next in the area, but we got quotas 🤪

**Review by laura-mannus — APPROVED** [review]

**Pull request merged**